src/main/java/edu/harvard/iq/dataverse/api/Admin.java

```diff
@@ -391,40 +381,9 @@ private Response deleteAuthenticatedUser(AuthenticatedUser au) {
         return ok("AuthenticatedUser " + au.getIdentifier() + " deleted. ");
 
     }  
+    
 
-    @POST
```

Comment:
Why is this section of code deleted? Doesn't seem to have anything to do with licenses.